enterprise/server/configure.md

```diff
@@ -239,7 +239,7 @@ ephox {
 
 {{site.requires_jsscwar_230v}}
 
-The Link-checker normally relies on the `HEAD` response. If `fallback-to-get` is `true`, the link-checker may issue a `GET` request in addition to the `HEAD` request to verify a link. When `true`, the Link checker can correctly identify working URLs that return non-standard `HEAD` replies. Enabling the `fallback-to-get` setting can lead to server performance issues and is set to `false` by default.
+The Link-checker normally relies on the `HEAD` response. If `fallback-to-get` is `true`, the link-checker may issue a `GET` request after the receiving a non-standard `HEAD` response to verify a link. When `true`, the Link checker can correctly identify working URLs that return non-standard `HEAD` replies. Enabling the `fallback-to-get` setting can lead to server performance issues and is set to `false` by default.
```

Comment:
after receiving or after the receiving of

Reply:
oops. I changed that sentence a few times to get it right, left an artifact behind...
Fixed by: https://github.com/tinymce/tinymce-docs/pull/1520/commits/51661117cf90c568349e6411bd427b9d3dd71be8